### DISCUSSION TITLE
Need help after resetting outline instance

### DISCUSSION THREAD

**Discussion by ovizii** [outline/outline#6662] [Self Hosting]
I was one of the few who were troubled by a [bug ](https://github.com/outline/outline/issues/5948) which duplicated content whenever saving a draft or publishing. I never managed to recover from that bug, [exporting all outline documents didn't work](https://github.com/outline/outline/issues/6094) either due to the same bug. 

So I manually exported each one of my content pages from outline. 
I then deleted all local data of outline as well as PostgreSQL. I added the slack keys back into my docker-compose.yml and fired up the stack to start fresh.

All worked well, I logged in with slack, I got the “Welcome to Outline” email checked all settings, enabled login via magic email link, shut down outline, removed the slack options from my compose and .env files and restarted outline.

I see the login with email option, enter my email and nothing happens. I checked the mail server's logs and the last sent email is the “Welcome to Outline” email. So for some reason, the magic login link is never sent. 

I set `      - LOG_LEVEL=info`  but I do not see anything about an email being sent out in the docker logs. 

It's like for some reason, between sending me the welcome email and a restart outline has forgotten how to send emails?

Please help!

**Comment by tommoor** [comment]
I think you've probably created an unexpected edge case here, in that you have a user that was created through Slack, but the originating Slack connection and integration has been removed from the configuration. This wasn't really a circumstance that has been accounted for.

In the database, find the `authentication_provider` record for Slack and change it to `enabled=false`, see if that helps.

**Reply by ovizii** [reply]
Thanks, I will try that but isn't that how one goes about using outline if one does not want any external authentication providers? 

Seeing that there is no way to get started with outline with the magic email link as the only authentication source? 

I guess I might log in via Slack, create an additional user via invite and email, then transfer admin right to that one and delete the Slack user. 

Will report back if that worked.

**Reply by ovizii** [reply]
I need to ask for help with this matter again. I did invite myself with a different email address and I received the invite via email. The link however looks a bit simple namely https://outline.mydomain.tld/?ref=invite-email - if I click the link, I access my outline instance, but there is no login method at all.

Any ideas?

![image](https://github.com/outline/outline/assets/837902/fae45976-2c3d-4254-9e2f-27dd5c75034b)

On the admin side all looks well. Email works fine, after all I received the invite I sent myself. Luckily I never logged out of my outline instance, or I'd have no way to log in again :-(
![image](https://github.com/outline/outline/assets/837902/e15b65c6-e86a-41ae-9539-d607557c541f)
